index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap" rel="stylesheet">
+
+
+    <link rel="stylesheet" href="style.css">
+
+
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div class="status-bar">
+            <div class="time-section">
+                <a href="#">
+                    <h2>9:41</h2>
+                </a>
+            </div>
+            <div class="right-section">
+                <img src="img/CellularConnection.png" alt="cellularconnection">
+                <img src="img/Wifi.png" alt="wifi">
+                <img src="img/Battery.png" alt="battery">
+            </div>
+        </div>
+        <div class="toolbar">
+            <div class="logo-section">
+                <h2>My Logo</h2>
+            </div>
+            <div class="icon-menu">
+                <img src="img/Union.png" alt="union">
+            </div>
+        </div>
+    </header>
+    <div class="body-section">
+        <div class="primary-text">
+            <h2>I'm Daniel</h2>
```

Comment:
# [OPTIONAL]
- [ ] Please consider changing the first `h2` tag in the "I'm Daniel" to an `h1` tag, this change is more for accessibility and semantics, as some reading tools will see the `h1` tag as more important.
![image](https://user-images.githubusercontent.com/73256642/133630853-e0442cdb-e8bc-4332-ae5e-1803b687b095.png)